docs/release_notes/release_1_1_0.rst

```diff
@@ -0,0 +1,20 @@
+:orphan:
+
+Release 1.1
+===========
+
+This release is work in progress.
+
+New features
+------------
+
+ - Exposing devices via TCP Stream has been made easier. It is now possible to define commands
+   that expose lambda-functions, named functions and data attributes (with separate read/write
+   patterns).
+
+Bug fixes and other improvements
+--------------------------------
+
+ - Virtually disconnecting devices via the control server now actually closes all network
```

Comment:
Would be useful to either have some imbedded examples or links to specific documentation for these new features. You already did that for the `pip install` thing in the `1_0_1` release notes.